### DISCUSSION TITLE
!!! emergency !!! not supported KMS on S3

### DISCUSSION THREAD

**Discussion by jawahiko** [trendmicro/cloudone-filestorage-plugins#143] [General]
表題の通りです。
KMS設定されたS3には隔離機能が働きません。。。DEVでは動作していたのに本番では動作しなくて困っていたところ、KMS暗号化がかかったS3では動作しないLambda関数になっていたことが発覚しました。。
python修正を試みてますが出来ず。。。急ぎ対応をお願いしたく。。

As the title says.
Quarantine function does not work for S3 with KMS settings. . . It worked in DEV, but when I was in trouble because it didn't work in production, I discovered that the Lambda function didn't work in S3 with KMS encryption. .
I'm trying to fix python, but I can't. . . I would like to ask for your urgent response. .

**Comment by felipecosta09** [comment]
Hi @jawahiko,

I think there is a PR #139 for that already waiting, the code is not 100% worked out, and since it belongs to somebody else I could not fix the code there. I do have a version with the code working already, let me know if you needed, I can put it someplace else so you can use it.
